### PR TITLE
Clean up authentication pages

### PR DESCRIPTION
**Pull request by tortila**
### Scope of changes:
- all authentication pages are served on `/accounts` URL path, there are no more equivalent pages on the `/admin` path 
- there is now explicit configuration for all authentication pages in `urls.py`, including which template is used. This replaces [the default implicit configuration of `django.contrib.auth.urls`](https://docs.djangoproject.com/en/4.1/topics/auth/default/#using-the-views)
- fixed redirects: after successful registration request the user stays on the registration screen, and after confirming their email address they're redirected to the login screen
- created new templates for: logout, password reset, password change
- organized all templates for authentication pages into the directory `apps > accounts > templates > auth`
- removed the usage of the waffle flag `access_old_admin_from_verification_list` - now the hyperlink to the admin panel is shown on the `/requests` page for all users

### To review:
- Copy of the confirmation/error messages that we display on various screens. Some of the wording in this PR is taken directly from [the default Django authentication templates](https://github.com/django/django/tree/main/django/contrib/admin/templates/registration), and some of them from older code.
- I'd appreciate a second pair of eyes testing functionally the login, logout, password reset and password change functionality, to verify that we don't have any dead hyperlinks
- Visual inspection of the newly created templates, feedback on the styling etc

#### Logout
logout
#### Password change requested
password_change
#### Password change done: new password set
password_change_done
#### Password reset requested
password_reset
#### Password reset done: activation link sent via email
password_reset_done
#### Password reset: assign a new password
password_reset_confirm
#### Password reset complete: new password set
password_reset_complete

### CODE REVIEW

**Comment by tortila** [comment]
[SPIKE: review user login/registration process and consider how to streamline](https://trello.com/c/c93LRj3e/113-spike-review-user-login-registration-process-and-consider-how-to-streamline)

**Comment by tortila** [comment]
[Create prompt for a new sign up user to check their email when signing up to log in.](https://trello.com/c/ZyYANMku/130-create-prompt-for-a-new-sign-up-user-to-check-their-email-when-signing-up-to-log-in)

File: apps/accounts/views.py [hunk 1]
```diff
@@ -100,20 +98,22 @@ def get_success_url(self, user=None):
...
+        return reverse("registration")
 
     def post(self, request, *args, **kwargs):
         form = self.get_form()
         if form.is_valid():
-            messages.add_message(
-                request, messages.INFO, "Check your email to activate the user"
+            messages.success(
+                request,
+                "Check your email inbox and open the activation link in order to complete the registration process.",
```
[the remark this answers is not in the document]
Comment (tortila) [reply apps/accounts/views.py]: thanks for the feedback - done in https://github.com/thegreenwebfoundation/admin-portal/pull/435/commits/8e9f707c2f4a76fc49b20c290f1b932da50a726f

**Comment by tortila** [comment]
[Change where we redirect people in the confirmation email when they sign up for the first time](https://trello.com/c/kp2rYC4s/131-change-where-we-redirect-people-in-the-confirmation-email-when-they-sign-up-for-the-first-time)

**Comment by tortila** [comment]
[Redirect logged in users to the new dashboard screen, not the full admin](https://trello.com/c/xpxRZ2tu/132-redirect-logged-in-users-to-the-new-dashboard-screen-not-the-full-admin)

**Review by mrchrisadams — APPROVED** [review]
Hi @tortila  - I'm v happy with this.

(We've reviewed this in person together)

**Pull request merged**